src/main/java/fr/novia/zaproxyplugin/ZAProxy.java

```diff
@@ -181,11 +186,11 @@
 	
 	/** The jdk to use to start ZAProxy */
 	private final String jdk;
-	
+
 	// Fields in fr/novia/zaproxyplugin/ZAProxy/config.jelly must match the parameter names in the "DataBoundConstructor"
 	@DataBoundConstructor
 	public ZAProxy(boolean autoInstall, String toolUsed, String zapHome, int timeoutInSec,
-			String filenameLoadSession, String targetURL, boolean spiderURL, boolean scanURL,
+			String filenameLoadSession, String targetURL, boolean spiderURL, boolean ajaxSpiderURL, boolean scanURL,
```

Comment:
Formally it causes a binary compatibility issues. It would be better to retain the original constructor and mark it as deprecated